recipes/mwclient/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: mwclient
+  version: "0.8.4"
+
+source:
+  fn: mwclient-0.8.4.tar.gz
+  url: https://pypi.python.org/packages/66/84/e99c39f3ab25a18eea9371d4e47ba79866d08f2e0ef3d70926f3ebd05c1b/mwclient-0.8.4.tar.gz
+  md5: 8532c9e0e0cbda4e1b6495c361d0310c
```

Comment:
Please update this to be sha256 instead of md5. [conda-forge prefers sha256](https://github.com/conda-forge/staged-recipes/blob/master/README.md#2-how-do-i-populate-the-hash-field)